src/exodus/sigmadb.cpp

```diff
@@ -176,10 +176,14 @@ CMPMintList::CMPMintList(const boost::filesystem::path& path, bool fWipe, uint16
     PrintToLog("Loading mint meta-info database: %s\n", status.ToString());
 
     this->groupSize = InitGroupSize(groupSize);
+    NotifyProcessedTransaction.connect(
```

Comment:
We should not use this way for core logic. The problems is if someone subscribe to `NotifyProcessedTransaction` before this class that mean their handler will get invoke before this class and they will expect that mints already stored in the database.